.circleci/config.yml

```diff
@@ -108,6 +108,29 @@ jobs:
                 name: Tag
                 command: |
                   ./.circleci/ci-docker-tag-op-geth-release.sh <<parameters.registry>>/<<parameters.repo>> $CIRCLE_TAG $CIRCLE_SHA1
+      - when:
+          condition:
+            equal: [optimism, << pipeline.git.branch >>]
```

Comment:
How does this image signer work? Is it supposed to sign all op-geth docker images that are based on merges into `optimism`?